code/modules/projectiles/updated_projectiles/guns/shotguns.dm

```diff
@@ -526,3 +526,50 @@ can cause issues with ammo types getting mixed up during the burst.
 	pump_delay = config.mhigh_fire_delay*2
 
 //-------------------------------------------------------
+
+//-------------------------------------------------------
+//SCOUT SHOTGUN
+
+/obj/item/weapon/gun/shotgun/merc/scout
+	name = "\improper ZX-76 assault shotgun"
+	desc = "The MIC ZX-76 Assault Shotgun, a semi-automatic combat shotgun capable of a double shot. Has a 10 round internal magazine."
+	icon_state = "zx-76"
+	item_state = "zx-76"
+	origin_tech = "combat=5;materials=4"
+	fire_sound = 'sound/weapons/gun_shotgun_automatic.ogg'
+	current_mag = /obj/item/ammo_magazine/internal/shotgun/scout
+	attachable_allowed = list(
+						/obj/item/attachable/bayonet,
+						/obj/item/attachable/reddot,
+						/obj/item/attachable/verticalgrip,
+						/obj/item/attachable/angledgrip,
+						/obj/item/attachable/gyro,
+						/obj/item/attachable/flashlight,
+						/obj/item/attachable/extended_barrel,
+						/obj/item/attachable/heavy_barrel,
+						/obj/item/attachable/compensator,
+						/obj/item/attachable/magnetic_harness,
+						/obj/item/attachable/attached_gun/flamer,
+						/obj/item/attachable/attached_gun/shotgun,) //if it can mount a flamer, why can't it mount a shotgun
+	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 10, "rail_y" = 21, "under_x" = 14, "under_y" = 16, "stock_x" = 14, "stock_y" = 16)
+
+/obj/item/weapon/gun/shotgun/combat/New()
```

Comment:
Even if tagged as outdated (turns out I forgot to press the button yesterday before nighttime, apologises), it should still be relevant since you just moved the code to another file.